recipes/openms-meta/meta.yaml

```diff
@@ -19,12 +19,11 @@ source:
 {% else %}
 source:
   url: https://github.com/OpenMS/OpenMS/releases/download/release%2F3.3.0/OpenMS-3.3.tar.gz # since we discarded the trailing 0 during the GHA we have to hard code this, fix it next release
-  md5: 7A5839E307801A7C9A02258F9DA61357
+  md5: 7a5839e307801a7c9a02258f9da61357
```

Comment:
1) Why don't you make a new release ? 3.3.1 
Updating an already released version is a bad practice.

2) Please switch to sha256